src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -175,6 +175,11 @@ let workbenchProperties: { [path: string]: IJSONSchema; } = {
 		'type': 'boolean',
 		'description': nls.localize('closeOnFileDelete', "Controls if editors showing a file should close automatically when the file is deleted or renamed by some other process. Disabling this will keep the editor open as dirty on such an event. Note that deleting from within the application will always close the editor and that dirty files will never close to preserve your data."),
 		'default': true
+	},
+	'workbench.fontAliasing': {
```

Comment:
Since this setting only seems to have an impact on macOS, you should check if your are on macOS and only contribute this setting on macOS (`platform.isMacintosh`).